### PR TITLE
new procedure for Docker on  M1 mac

### PR DESCRIPTION
**Pull request by womullan**

### CODE REVIEW

File: doc/procedures/dockermacm1.rst [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+Running the container
+=====================
+
+The `develop` tag indicates the most recent container that should be used for development.
+The development container is `develop-env`.
+On Mac M1 to avoid a warning add `--platform linux/amd64`.
+Naming the container is useful for logging in and finding it later.
+The '-v' option may be used to mount a local directory into the container - keeping all you checkouts in a folder outside is useful if you want to use an IDE for editing.
+
+So a command to spin up a container i and put you on the command prompt would look like (with output) :
```
Comment (pkubanek) [inline doc/procedures/dockermacm1.rst:36]: Drop " i "?

File: doc/procedures/dockermacm1.rst [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+.. figure:: ../images//DockerSettingD.png
+   :name: fig-dockerd
+   :alt: Docker Development Settings
+
+
+Running the container
+=====================
+
+The `develop` tag indicates the most recent container that should be used for development.
+The development container is `develop-env`.
```
Comment (pkubanek) [inline doc/procedures/dockermacm1.rst:31]: What about stable containers - cycles? That's handy when you need containers running on summit. https://confluence.lsstcorp.org/display/LSSTCOM/Software+Upgrades lists cycles.

https://hub.docker.com/r/lsstts/develop-env/tags lists develop-env tags on Dockerhub. However, as the images were moved to own server, one now needs to retrieve cCCCC.VVV, where CCCC is cycle number (0029) and VVV is version (000 for example). So to get proper c0029, I use:

ts-dockerhub.lsst.org/develop-env:c0029.000

**Review by pkubanek — COMMENTED** [review]
Please take a look at my comments. I am lost in the current docker images location, the values I put in are the current cycle qualified guess, based on a conversation with the team.

File: doc/procedures/dockermacm1.rst (hunk 1 above)
Comment (couger01) [inline doc/procedures/dockermacm1.rst:36]:
```suggestion
So a command to spin up a container ``i`` and put you on the command prompt would look like (with output) :
```

File: doc/procedures/dockermacm1.rst (hunk 2 above)
Comment (couger01) [inline doc/procedures/dockermacm1.rst:31]:
```suggestion
The development container is `develop-env`.
You can find cycle tags on ts-dockerhub.lsst.org with ``ts-dockerhub.lsst.org/develop-env:c0029.000``
```

**Review by couger01 — APPROVED** [review]
LGTM but a few minor suggestions based on Petr's feedback.

**Comment by couger01** [comment]
Also don't forget to rebase on main.

File: doc/procedures/dockermacm1.rst (hunk 2 above)
Comment (womullan) [reply doc/procedures/dockermacm1.rst:31]: ok put a version of this in the text

Comment (womullan) [reply doc/procedures/dockermacm1.rst:31]: I only see a c0028.004

**Comment by wvreeven** [comment]
I made changes to the TSSW Dev Guide today and found that the code blocks that were added in this PR didn't compile well so I fixed that along the way. This is part of the PR for DM-38860.

**Pull request merged**